### PR TITLE
added beautiful image

### PR DESCRIPTION
**Pull request by kumankai**

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -8,8 +9,9 @@ function App() {
           The Official Sheridan College Computer Science Club Website!
         </h1>
         <p className="p-text">
-          Yeah this site is empty. Get to codin club members ;)
```
Comment (AmirGhiassian) [inline src/App.tsx:11]: Why was the ")" removed 😭
It was a winkyyy faceeeeee

**Review by AmirGhiassian — CHANGES_REQUESTED** [review]
Looks lovely, maybe just one thing. Change the text since now we have a pull request and maybe we can get more users to commit to the repository

File: src/App.tsx (hunk 1 above)
Comment (kumankai) [reply src/App.tsx:11]: Sorry, thought it was a type

**Review by AmirGhiassian — APPROVED** [review]
;)

**Pull request merged**